troubleshooting.md

```diff
@@ -0,0 +1,90 @@
+By default, the single `master` container will be named `dcos-docker-master1`.
+
+To get a shell on this node, run:
+
+```
+$ docker exec -it dcos-docker-master1 bash
+```
+
+Once on the node, list `systemd` units:
+
+```
+[root@dcos-docker-master1 /]# systemctl list-units
+...
+dbus.socket                         loaded active     running         D-Bus System Message Bus Socket
+systemd-fail.service                loaded failed     exited          Journal Audit Socket
+systemd-journald-dev-log.socket     loaded active     running         Journal Socket (/dev/log)
+systemd-journald.socket             loaded active     running         Journal Socket
+basic.target                        loaded active     active          Basic System
+dcos.target                         loaded active     active          dcos.target
+local-fs.target                     loaded active     active          Local File Systems
+...
+```
+
+Choose a `systemd` unit with the `ACTIVE` status `exited` and check its status.
+In the above example, the unit is named `systemd-fail.service`.
+
+```
+[root@dcos-docker-master1 /]# systemctl status systemd-fail
+```
+
+Then get the logs for this unit:
+
+```
+[root@dcos-docker-master1 /]# journalctl -xefu systemd-fail
+```
+
+On hosts without `systemd` such as macOS and legacy versions of Ubuntu, an unsupported Mesos setting is used.
+See "Non-systemd host" in [`README.md`](./README.md) for details.
+
+## `dcos-spartan` does not start
+
+For the `dcos-spartan` service to start successfully, make sure that
+you have dummy net driver support (`CONFIG_DUMMY`) enabled in your kernel.
+Most standard distribution kernels should have this by default. On some
+older kernels you may need to manually install this module with
+`modprobe dummy` before starting the container cluster.
+
+## Docker out of space
+
+There are multiple symptoms and fixes for Docker being out of space.
+If an error which suggests that Docker is out of space presents, try the following command:
+
+```
+docker volume prune
+```
+
+It is possible that DC/OS services will fail to start and an error similar to the following will be reported in `journalctl`.
+
+```
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: Traceback (most recent call last):
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: File "/opt/mesosphere/packages/exhibitor--72d9d8f947e5411eda524d40dde1a58edeb158ed/usr/exhibitor/start_exhibitor.py", l
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: """)
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: File "/opt/mesosphere/packages/exhibitor--72d9d8f947e5411eda524d40dde1a58edeb158ed/usr/exhibitor/start_exhibitor.py", l
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: f.write(contents)
+Jun 26 22:32:44 dcos-docker-master1 start_exhibitor.py[6242]: OSError: [Errno 28] No space left on device
+```
+
+On Docker for Mac, this can be fixed by deleting the disk image.
```

Comment:
Should we add a warning here that this obliterates all locally stored Docker images, including those unrelated to DC/OS (I don't know anything about Docker on a Mac, but this is what I believe to have understood from here...)?